docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.mdx

```diff
@@ -4,6 +4,10 @@
 
 This article is a comprehensive command reference for Stencil CLI, BigCommerce's powerful theme development and deployment tool. For installation instructions for your OS, see [Installing Stencil CLI](/stencil-docs/installing-stencil-cli/installing-stencil). For more information on BigCommerce's Stencil Theme Engine, see [About Stencil](/stencil-docs/getting-started/about-stencil). Continue reading below for detailed information on each Stencil CLI command and option.
 
+<Callout type="warning">
+BigCommerce is currently targeting 11/1/2023 to sunset node-sass. To ensure that your storefront is up to date, use the latest node version (Node 18) in Stencil CLI and install the automated script to resolve [incompatible SCSS directives](/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#incompatible-scss-directives), which can cause issues with the styling of your storefront.
```

Comment:
could you use the same wording as above https://github.com/bigcommerce/dev-docs/pull/1855/files#diff-d653d450312b9b922608cb1e4495dbac19f5a10cc1d79e7d54c5572ab4187ddeR4